tests/csv/1.0/samplesheet.csv

```diff
@@ -0,0 +1,3 @@
+sample,fastq_1,fastq_2,strandedness
```

Comment:
this should be in the test data repo

Reply:
I mean all test data should be hosted in `nf-core/test-datasets`, not in the local repo